resources/js/common.js

```diff
@@ -283,3 +282,40 @@ function postwith (to,p) {
   document.body.removeChild(myForm) ;
 }
 
+
+function isValidDate(dateString)

```

Comment:
@remocrevo , nowadays it seems hard to find a working validation plugin for jquery 1.4.4 which is loaded by the Resource  pages.  On the other hand, the newest validation plugin can't handle leap years well as opposed to the validateDate() function added by this PR a while ago, though this is not a game changer at all.  Maybe we also need to discuss jQuery versions in the upcoming SC meeting. For now, as mentioned in my early comment, I completely drop out the date   validation function and we can re-add it later on separately. By the way, would you mind testing and merging this PR before this PR becomes conflicting again? Any direct editing and committing upon problems found is welcome. Thank you.